### PR TITLE
Valentin

### PR DESCRIPTION
**Pull request by Atadeno**
Fixed linting by adding exceptions and changing code for some errors
Updated tutorials with new images for the pipeline
Structure test with pre/pos test
Added precommit for husky

### CODE REVIEW

File: web-component/inputRT.js [hunk 1]
```diff
@@ -1,5 +1,5 @@
-(function(){
```
Comment (ericsengithub) [inline web-component/inputRT.js]: please add this function back

File: web-component/CoreHello.js [hunk 2]
```diff
@@ -3,174 +3,187 @@
...
  */
 class CoreHello extends HTMLElement {
-    /**
-     * get rainbow() 
-     * Check if rainbow exists in HTML.
-     * Returns: True or False 
-     */
-    get rainbow() {
-      return this.hasAttribute('rainbow');
+  /**
```
Comment (ericsengithub) [inline web-component/CoreHello.js]: please make the spacing 4 spaces

File: web-component/inputRT.js (hunk 1 above)
Comment (ericsengithub) [reply web-component/inputRT.js]: its necessary to keep the scope of the file private

File: web-component/inputbox-rt-default-style.css [hunk 3]
```diff
@@ -1,27 +0,0 @@
-/* default styles */
```
Comment (ericsengithub) [inline web-component/inputbox-rt-default-style.css]: why was this deleted?

File: web-component/CoreHello.js [hunk 4]
```diff
@@ -3,174 +3,187 @@
...
+
+  setLanguages() {
+    return {
+      "en": "Hello World",
+      "ar": "مرحبا بالعالم",
+      "es": "Hola Mundo",
+      "fr": "Bonjour le monde",
+      "zh": "你好"
+    };
+  }
```
Comment (ericsengithub) [inline web-component/CoreHello.js]: did you make sure the langues still work?

**Comment by Atadeno** [comment]
Trying again
